addons/config/run.tanzu.vmware.com/v1alpha3/kappcontrollerconfig.yaml

```diff
@@ -0,0 +1,24 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: run.tanzu.vmware.com/v1alpha3
+kind: KappControllerConfig
+metadata:
+  name: #@ data.values.TKR_VERSION
+  namespace: #@ data.values.GLOBAL_NAMESPACE
+spec:
+  namespace: test-ns
+  kappController:
+    globalNamespace: #@ data.values.GLOBAL_NAMESPACE
+    deployment:
+      apiPort: 10100
+      tolerations:
+        - key: CriticalAddonsOnly
+          operator: Exists
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/master
+        - effect: NoSchedule
+          key: node.kubernetes.io/not-ready
+        - effect: NoSchedule
+          key: node.cloudprovider.kubernetes.io/uninitialized
+          value: "true"
+
```

Comment:
Could we default other vars as well?
```
      concurrency: 4
      hostNetwork: true
      metricsBindAddress: "0"
      priorityClassName: system-cluster-critical
```

Reply:
These have the same value as the default of config API. Do we need to reiterate on them?

Reply:
Lets be consistent. If we are defining some attributes, why not all.

Reply:
Lets be consistent. If we are defining some attributes, why not all.

Reply:
The attributes I defined have different default values, that's the reason for calling them out explicitly.